eco.Rmd

```diff
@@ -17,7 +20,9 @@ structure:
 7. discussion: what could be done better or alternatives and again emphasizing that the methods shown
 finally, point to books on ecological modeling especially emphasizing Alain Zuur's books
 
+
 ## Background
+
 Fog oases are one of the most fascinating vegetation formations I have ever encountered. 
```

Comment:
I think this is too personal: if it's aiming to be a text book I think we should use a more consistent text-booky style. There are times in r4ds when they use "I" and it does not read well for me. So I suggest you use "we" or (better IMO) the passive voice here.

Reply:
Ok, will change.